src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,167 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
```

Comment:
copy-paste from java.util.HashMap is not acceptable. try to solve task all by yourself, simplifying some moments